src/components/TodoComponents/Todo.js

```diff
@@ -5,7 +5,7 @@ const Todo = props => {
 	return (
 		<ul>
 			{props.dailyTodos.map(input => {
-				return <li key={Math.random()}>{input}</li>;
+				return <li key={Math.random()}>{input.task}</li>;
```

Comment:
nice use of Math.random() for the key also remember that Date.now() and if you are doing any larger or production code maybe install an npm module uuid and use uuid.v4()